client/src/components/mapping/FhirMappingPanel/ResourceSelector/Drawer/index.tsx

```diff
@@ -52,7 +46,7 @@ const Drawer = ({ isOpen, onCloseCallback }: Props): ReactElement => {
   const [label, setLabel] = React.useState('');
   const [pkTable, setPkTable] = React.useState('');
   const [pkColumn, setPkColumn] = React.useState('');
-  const [filters, setFilters] = React.useState([] as Filter[]);
+  const [filters, setFilters] = React.useState([] as Filters[]);
```

Comment:
nit + I know it's not you: could we rename `Filters` -> `Filter`?

Reply:
done